### ISSUE TITLE
Wait! What Does This Mean!! (IMPORTANT PLEASE READ)

### ISSUE THREAD

**Issue by bigguystevieb1** [glixzzy/blooket-hack#1426]
![Screenshot 2022-02-23 1 13 30 PM](https://user-images.githubusercontent.com/99492003/155381484-7ea007ca-5095-4690-b81e-5e5567d1d752.png)

**Comment by CHAKAMAN12** [comment]
hmmmm?

**Comment by CHAKAMAN12** [comment]
Very interesting...

**Comment by bigguystevieb1** [comment]
This has to mean something... Maybe he is keeping a secret from us, If so this is very shady. In my opinion..

**Comment by CHAKAMAN12** [comment]
idk but maybe it is a good thing??

**Comment by bigguystevieb1** [comment]
How would this be a good thing?

**Comment by CHAKAMAN12** [comment]
honestly i don't know what i am saying🙄

**Comment by bigguystevieb1** [comment]
This is very shady. Im honestly going to try to get to the bottom of this.

**Comment by CHAKAMAN12** [comment]
> This is very shady. Im honestly going to try to get to the bottom of this.

Please do that. It might lead to something...

**Comment by bigguystevieb1** [comment]
> > This is very shady. Im honestly going to try to get to the bottom of this.
> 
> Please do that. It might lead to something...

Yes. I will do the best that I can do.

**Comment by CHAKAMAN12** [comment]
i will try to find stuff every now and then as well, if i can.

**Comment by bigguystevieb1** [comment]
I will let you know with any updates.

**Comment by bigguystevieb1** [comment]
I'm now curious to who this "FAKE" glixzzy is.

**Comment by bigguystevieb1** [comment]
I have to go now. I will be back some time later today.

**Comment by CHAKAMAN12** [comment]
Why would someone bother making another account?? There must be something behind this annoymous person.

**Comment by ForkAndSpoonIt** [comment]
it says he isnt associated. If that is true, they wouldnt use the same account. they banned him and someone took the name

**Comment by CHAKAMAN12** [comment]
Why would they bother taking the name though. I'm just saying it seems very suspicious to just do that.

**Comment by ForkAndSpoonIt** [comment]
I dont know. but the origional annot legally take the name. i dont think the code changed though. not like an ip grab

**Comment by CHAKAMAN12** [comment]
yeah i guess

**Comment by Blooket-dude** [comment]
hey can we make a google docs and type in case this gets delted?

**Comment by ForkAndSpoonIt** [comment]
Im in class. cant add anyone outside of district

**Comment by CHAKAMAN12** [comment]
same

**Comment by CHAKAMAN12** [comment]
plus this probably will get deleted

**Comment by CHAKAMAN12** [comment]
> hey can we make a google docs and type in case this gets delted?

If i got found out by my teacher i would be done for.

**Comment by CHAKAMAN12** [comment]
btw this isn't really on the topic of anything on this but why was discussions removed??

**Comment by Blooket-dude** [comment]
ok thats fine

**Comment by Blooket-dude** [comment]
well we can make our own project thing and talk there if ou guys want to.

**Comment by CHAKAMAN12** [comment]
what exactly do you mean by that?

**Comment by Blooket-dude** [comment]
I mean we can just make a respostory and then we can create and issue and talk there. jsut in case this gets deleted

**Comment by CHAKAMAN12** [comment]
guys i don't really understand 99.9% of what this means but why is these contributions have some from like 2021 before this glixzzy account was made??
![Screenshot 2022-02-23 2 03 42 PM](https://user-images.githubusercontent.com/98620341/155390638-67d84ea7-282c-4a1b-ad00-27bb8b8d7011.png)

**Comment by Spearmintiscool** [comment]
Its says they joined 10 hours ago on there, which means someone made account acting like them or its glixzzy.

**Comment by Blooket-dude** [comment]
but why would glixzzy do it again?

**Comment by CHAKAMAN12** [comment]
yeah thats why i am still confused by it but it still has like all the past info about the account

**Comment by Spearmintiscool** [comment]
I mean 19 hours

**Comment by Blooket-dude** [comment]
yes its weird really really weird

**Comment by CHAKAMAN12** [comment]
But this account was created 19 to 20 hours aqgo but it has all that info about it before this account was created.

**Comment by Blooket-dude** [comment]
I know that what im seeing as well.

**Comment by CHAKAMAN12** [comment]
We could try to crack this case but it will be very hard unless we have someone who knows a bit too much

**Comment by Blooket-dude** [comment]
a bit too much of what?

**Comment by Spearmintiscool** [comment]
Maybe glixzzy got banned for a few days but github unbanned glixzzy but since glixzzy has gotten the account again the computer sytem thinks that its a new account but it keeps everything else.

**Comment by CHAKAMAN12** [comment]
that could be true but this glixzzy said how they were not associated

**Comment by CHAKAMAN12** [comment]
which would mean it isn't him

**Comment by CHAKAMAN12** [comment]
from what my brain can think at least

**Comment by Spearmintiscool** [comment]
I think it is glizxxy but when when they got there account banned and when it got unbanned it didn't reset his contributions but it thinks it's a new account.

**Comment by Spearmintiscool** [comment]
The computer system and cpu thought it was a new account but it was wasn't actually.

**Comment by CHAKAMAN12** [comment]
it could be very true but first lets gather a bit more information before we come to the final conclusion

**Comment by CHAKAMAN12** [comment]
anyways i have to go i will see everyone tomorrow if this thing stays up.

**Comment by Blooket-dude** [comment]
yes agreed

**Comment by Greenlio** [comment]
Listen up idiot clowns, he was forced to delete it after he was threatened with a heavy lawsuit from Ben's dad. He got his account back from GitHub and that's why it says what it says.
